samples/system-test/microphonestream.test.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2016, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require(`path`);
+//const storage = require(`@google-cloud/storage`)();
```

Comment:
There are multiple commented out variables here and throughout the file. Could you provide some insight as to why? Are these left over from testing/playing you were doing? Can I expect these to be changed or removed?